docs/downloader.rst

```diff
@@ -0,0 +1,169 @@
+
+.. -*- coding: utf-8 -*-
+
+.. _downloader:
+
+Downloader Operation
+====================
+
+Audience: developers working on magic-folder itself
+
+**Note:** some of this document describes features that aren't yet implemented; those are noted with (Not implemented).
```

Comment:
I've not yet read this, but it would be good to mark which parts are implemented, and which are aspirational. Partly to make seeing if the code below matches the description here.

Reply:
Parts that don't say "(Not implemented)" are implemented.